utils.py

```diff
@@ -498,3 +498,15 @@ def get_template_dir_prefix():
     template_path = ('/templates/head' if feconf.IS_MINIFIED
                      or not feconf.DEV_MODE else '/templates/dev/head')
     return '%s%s' % (get_asset_dir_prefix(), template_path)
+
+
+def convert_unicode_to_str(state_name):
+    """Converts the given unicode string to a string. If the string is not
```

Comment:
It's not guaranteed that the given string is unicode. It may also be str. So maybe rename this function convert_to_str() and update the docstring accordingly.